client/my-sites/domains/domain-management/domain-connect-mapping/index.jsx

```diff
@@ -198,7 +205,12 @@ class DomainConnectMapping extends React.Component {
 				( data ) => {
 					const success = get( data, 'success', false );
 					const syncUxUrl = get( data, 'sync_ux_apply_url', null );
```

Comment:
Minor comment, these lines were already like this, but they could be replaced by `data?.success` and `data?.sync_ux_apply_url` to reduce our use of `lodash`. There's another use of `get` in line 51 that could also be replaced.